concent_api/api-integration-force-accept-or-reject-results-test.py

```diff
@@ -109,181 +103,147 @@ def report_computed_task(timestamp = None, task_to_compute = None):
+    (cluster_url, patterns) = parse_arguments()
+    cluster_consts = get_protocol_constants(cluster_url)
+    print_protocol_constants(cluster_consts)
+
+    test_id = str(random.randrange(1, 100000))
+    tests_to_execute = get_tests_list(patterns, list(globals().keys()))
+
+    print("Tests to be executed: \n * " + "\n * ".join(tests_to_execute))
+    print()
+
+    execute_tests(
+        tests_to_execute=tests_to_execute,
+        objects=globals(),
+        cluster_url=cluster_url,
+        test_id=test_id,
+        cluster_consts=cluster_consts
+    )
+
+    if count_fails.get_fails() > 0:
+        count_fails.print_fails()
+    print("END")
+
+
+@count_fails
+def test_case_2d_requestor_rejects_subtask_results(cluster_consts, cluster_url, test_id):
+    # Test CASE 2D + 3 + 4B + 5. Requestor sends ForceSubtaskResultsResponse with SubtaskResultsRejected
+    current_time = get_current_utc_timestamp()
+    (subtask_id, task_id) = get_task_id_and_subtask_id(test_id, '2D')
     api_request(
         cluster_url,
         'send',
         PROVIDER_PRIVATE_KEY,
         CONCENT_PUBLIC_KEY,
         force_subtask_results(
-            timestamp = timestamp_to_isoformat(current_time),
-            ack_report_computed_task = ack_report_computed_task(
-                timestamp = timestamp_to_isoformat(current_time - (cluster_consts.subtask_verification_time * 1.4)),
-                report_computed_task = report_computed_task(
-                    task_to_compute = task_to_compute(
-                        timestamp = timestamp_to_isoformat(current_time - cluster_consts.subtask_verification_time * 1.5),
-                        compute_task_def = compute_task_def(
-                            task_id     = task_id,
-                            subtask_id  = subtask_id,
-                            deadline    = current_time - (cluster_consts.subtask_verification_time),
+            timestamp=timestamp_to_isoformat(current_time),
+            ack_report_computed_task=ack_report_computed_task(
+                timestamp=timestamp_to_isoformat(current_time - (cluster_consts.subtask_verification_time)),
+                report_computed_task=report_computed_task(
+                    task_to_compute=task_to_compute(
+                        timestamp=timestamp_to_isoformat(current_time - cluster_consts.subtask_verification_time * 1.5),
+                        compute_task_def=compute_task_def(
+                            task_id=task_id,
+                            subtask_id=subtask_id,
+                            deadline=current_time - (cluster_consts.subtask_verification_time),
                         )
                     )
                 )
             )
         ),
-        headers = {
-            'Content-Type':                     'application/octet-stream',
-            'temporary-account-funds':          'True'
+        headers={
+            'Content-Type': 'application/octet-stream',
+            'temporary-account-funds': 'True'
```

Comment:
Are you aware that this is part you said is not needed in https://github.com/golemfactory/concent/pull/213/files ?

Reply:
it is still needed, because @kbeker 's changes are not yet merged. It should be removed as a part of @kbeker 's PR